packages/esbuild-plugin/package.json

```diff
@@ -1,40 +1,71 @@
 {
     "name": "@datadog/esbuild-plugin",
     "packageManager": "yarn@4.0.2",
+    "version": "0.0.13-4",
     "license": "MIT",
     "author": "Datadog",
+    "description": "Datadog ESBuild Plugin.",
+    "keywords": [
+        "datadog",
+        "esbuild",
+        "bundler",
+        "plugin",
+        "unplugin"
+    ],
+    "homepage": "https://github.com/DataDog/build-plugin#readme",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/DataDog/build-plugin",
+        "directory": "packages/esbuild-plugin"
+    },
     "main": "./dist/src/index.js",
+    "module": "./dist/src/index.mjs",
     "exports": {
         ".": "./src/index.ts",
         "./*": "./src/*.ts"
     },
     "publishConfig": {
         "access": "public",
+        "types": "./dist/src/index.d.ts",
         "exports": {
             "./package.json": "./package.json",
-            ".": "./dist/src/index.js",
-            "./*": "./dist/src/*.js"
+            ".": {
+                "import": "./dist/src/index.mjs",
+                "require": "./dist/src/index.js",
+                "types": "./dist/src/index.d.ts"
+            }
         }
     },
     "files": [
         "dist"
     ],
     "scripts": {
-        "build": "yarn clean && tsc",
+        "build": "yarn clean && rollup --config rollup.config.mjs",
```

Comment:
If the code is built by `rollup`, you way want to add `"moduleResolution": "bundler"` & `"module": "NodeNext"` in your TSConfig.json

Reply:
Used `"preserve"` as discussed, except for the tests package.